api/guids/views.py

```diff
@@ -33,10 +35,29 @@ class GuidRedirect(JSONAPIBaseView):
     view_category = 'guids'
     view_name = 'guid-detail'
 
+    serializer_class = GuidSerializer
+
+    @staticmethod
+    def should_resolve(request):
+        resolve = request.query_params.get('resolve')
+        return resolve is None or is_truthy(resolve)
```

Comment:
Are we "resolving" by default? I was under the impression that resolving would be opt-in, but I'm fine with this as long as @brianjgeiger is good with it.